tests/testthat/test-tidyme.R

```diff
@@ -0,0 +1,86 @@
+#' @title tidyme
+#' @section Last updated by:
+#' Tim Treis
+#' @section Last update date:
+#' 19-MAR-2021
+
+# Specifications ---------------------------------------------------------------
+
+#' T1.  The function properly converts "messy output" into tidy tibbles
```

Comment:
Properly is vague.
T1. The function tidies up an associated object
T1.1 The default method returns a tibble, tidies with broom
T1.2 The default method does not affect the content of the object
T1.3 The S3 method, associated with a survival object, returns a ...
T2 The S3  method, associated with a survival object, transforms the survival object into a data frame
T2.1 Which information is retained and which is not??
T2.X .....